packages/guides/src/Compiler/CompilerContext.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @link https://phpdoc.org
+ */
+
+namespace phpDocumentor\Guides\Compiler;
+
+use phpDocumentor\Guides\Nodes\ProjectNode;
+
+class CompilerContext
+{
+    public function __construct(
```

Comment:
Now we have a Compiler context, maybe we should move the document in here as well.